Sources/MlemMiddleware/Custom API Models/Extensions/APIPersonView+Extensions.swift

```diff
@@ -7,4 +7,12 @@
 
 import Foundation
 
-extension ApiPersonView: Person2ApiBacker {}
+extension ApiPersonView: Person2ApiBacker {
+    public var admin: Bool {
+        if let admin = self.isAdmin ?? self.person.admin {
+            return admin
+        }
+        assertionFailure()
+        return false
```

Comment:
*nit*

Perhaps more elegant to solve with a `guard`:

```suggestion
        guard let admin = self.isAdmin ?? self.person.admin else {
            assertionFailure("Could not determine admin status")
            return false
        }
        return admin
```